source/module_base/parallel_global.cpp

```diff
@@ -14,6 +14,8 @@
 #include "module_base/global_function.h"
 #include "module_base/parallel_common.h"
 #include "module_base/parallel_reduce.h"
+#include "module_parameter/parameter.h"
+// #include "module_base/tool_quit.h"
```

Comment:
useless include ?

Reply:
@Qianruipku He planned to replace the exit(1) which will cause memory not being released with a WARNING_QUIT call, but due to the complex dependence between modules of unittests, he gave up after many trials, eft this #include line there, hoping will do it in the future.